shaal/DrupalPod
# Improve README + directory structure

# The Problem/Issue/Bug

This PR contains mostly file location/name changes and name changes.
From `.gitpod/drupal` to `.gitpod/drupal/ssh/`

The main update is that README now has meaningful content.

## How this PR Solves The Problem
README would help people get some understanding how to use this project.

## Manual Testing Instructions
Make sure the repo works in Gitpod -
* <a href="https://gitpod.io/#https://github.com/shaal/DrupalPod/pull/5"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
* Run `ddev drush si standard -y` and confirm a standard Drupal website was installed successfully.


## Related Issue Link(s)

## Release/Deployment notes



<a href="https://gitpod.io/#https://github.com/shaal/DrupalPod/pull/5"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
